xbmc/windowing/win10/WinSystemWin10.cpp

```diff
@@ -650,11 +651,40 @@ bool CWinSystemWin10::MessagePump()
   return m_winEvents->MessagePump();
 }
 
-int CWinSystemWin10::GetGuiSdrPeakLuminance() const
+/*!
+ * \brief Max luminance for GUI SDR content in HDR mode.
+ * \return Max luminance in nits, lower than 10000.
+*/
+unsigned int CWinSystemWin10::GetGuiSdrPeakLuminance() const
 {
   const auto settings = CServiceBroker::GetSettingsComponent()->GetSettings();
 
-  return settings->GetInt(CSettings::SETTING_VIDEOSCREEN_GUISDRPEAKLUMINANCE);
+  // use cached system value as this is called for each frame
+  if (settings->GetBool(CSettings::SETTING_VIDEOSCREEN_USESYSTEMSDRPEAKLUMINANCE) &&
+      m_validSystemSdrPeakLuminance)
+    return m_systemSdrPeakLuminance;
+
+  // Max nits for 100% UI setting = 1000 nits, < 10000 nits, min 60 nits for 0%
+  int guiSdrPeak = settings->GetInt(CSettings::SETTING_VIDEOSCREEN_GUISDRPEAKLUMINANCE);
+  return 10000 / ((100 - guiSdrPeak) * 1.56f + 10);
```

Comment:
I'm not sure about this change of scale: 

Original intention was 0% ---> 80-90 nits 

Since no one wants the GUI in HDR less bright than in SDR. 
The complaints were all in this direction: "the GUI in HDR is not very bright"

It is also a subjective matter because if the video is very bright, we perceive the GUI as less bright than it really is (our brain is based on comparisons, it is not a colorimeter 😊)

Reply:
Maybe my situation is special with OLED adjusted for a dim-lighted room, the GUI and the subs are too bright.
I prefer grayish subs in bright scenes to blinding subs in a dark scene (OLED does perfect black).
Someone viewing in direct sunlight would want extra bright instead, I understand.

In my case the manual setting has to be 80 nits or a bit less to match the SDR levels. I figured a bit of safety room wouldn't hurt, but I'll settle for 80 and understand the reasoning.

GUI brightness relative to the video content could be interesting actually, a bit like ambient lighting, but inside the screen :-)